File: app/src/main/kotlin/com/spacex/ui/emptyView/EmptyView.java
package com.spacex.ui.emptyView;

import android.content.Context;
import android.util.AttributeSet;
import android.view.LayoutInflater;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.constraintlayout.widget.ConstraintLayout;
import androidx.databinding.DataBindingUtil;
import com.spacex.ui.R;

public class EmptyView extends ConstraintLayout {

  public EmptyView(@NonNull Context context) {
    this(context, null);
  }

  public EmptyView(@NonNull Context context, @Nullable AttributeSet attrs) {
    this(context, attrs, 0);
  }

  public EmptyView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
    super(context, attrs, defStyleAttr);
    DataBindingUtil.inflate(LayoutInflater.from(context), R.layout.empty_view, this, true);
  }
}

File: app/src/main/kotlin/com/spacex/ui/customViews/EmptyView.kt
package com.spacex.ui.customViews

import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import androidx.constraintlayout.widget.ConstraintLayout
import androidx.databinding.DataBindingUtil
import androidx.databinding.ViewDataBinding
import com.spacex.ui.R

internal class EmptyView @JvmOverloads constructor(
        context: Context,
        attrs: AttributeSet? = null,
        defStyleAttr: Int = 0) : ConstraintLayout(context, attrs, defStyleAttr) {

    init {
        DataBindingUtil.inflate<ViewDataBinding>(
                LayoutInflater.from(context), R.layout.empty_view, this, true)
    }
}
